music21/harmony.py

```diff
@@ -245,28 +246,30 @@ def _updatePitches(self):
         '''
         return
 
-    def _updateBasedOnXMLInput(self, keywords):
+    def _updateBasedOnXMLInput(self, keywords, *, parseInversion=True):
         '''
         This method must be called twice, once before the pitches
         are rendered, and once after. This is because after the pitches
         are rendered, the root() and bass() becomes reset by the chord class
         but we want the objects to retain their initial root, bass, and inversion
         '''
         for kw in keywords:
+            if not keywords[kw]:
+                continue
```

Comment:
I think that all the keywords should be explicit if parseInversion is going to be added.  It seems weird to have one dict[keywords] and one lone actual keyword.

Reply:
True, that could be confusing. Easier to just remove `parseInversion`!